src/integer/mat_z/from.rs

```diff
@@ -80,9 +77,10 @@ impl FromStr for MatZ {
     /// # Errors and Failures
     /// - Returns a [`MathError`] of type [`StringConversionError`](MathError::StringConversionError)
     ///     - if the matrix is not formatted in a suitable way,
-    ///     - if the number of entries in rows is unequal,
-    ///     - if an entry contains a Nul byte, or
+    ///     - if the number of rows or columns is too large (must fit into i64),
+    ///     - if the number of entries in rows is unequal, or
     ///     - if an entry is not formatted correctly.
+    ///     - For further information see [`Z::from_str`].
```

Comment:
this formatting only relates to Z::from_str -> hence the statement "For further information..." should be on that dedicated bullet point. Otherwise it might be confusing